main.py

```diff
@@ -54,7 +54,7 @@ def main():
                     if case == 0:
                         first_name = str(input("Please enter your first name: "))
 
-                        if invalid_string(first_name):
+                        if not is_valid_string(first_name):
```

Comment:
Adding the "not" keyword changes the logic so that only invalid strings are accepted.